### DISCUSSION TITLE
Next update of scipy module

### DISCUSSION THREAD

**Discussion by tirumaleshn2458** [pyodide/pyodide#3930] [General]
Hi, I couldn't access the scipy of current version in pyodide with the current version of sckikit-learn in pyodide. Please update the scipy version. Thank you

**Comment by ryanking13** [comment]
We updated the Scipy version to 1.10.1 (latest) two months ago, but we haven't released Pyodide including it yet. You can use the main branch of Pyodide until we release the next version of Pyodide.
